---
title: |
  | Genetic endowments
  | and wealth inequality
subtitle: |
  Daniel Barth, Nicholas W. Papageorge, Kevin Thom\
  *Journal of Political Economy*, 2020, vol. 128, No.4
author: "伊藤成朗"
engine: knitr
execute:
  echo: false
  freeze: auto
format:
  revealjs:
    standalone: true
    embed-resources: true
    slide-number: true
    incremental: true
    width: 1600
    height: 900
    preview-links: auto
    logo: GrootbergGiraffeHead.jpg
    footer: <http://seiroito.github.io/QuartoFiles/posts/BarthPapageorgeThom/BPT.html>
    theme: [sky]
    linkcolor: red
    code-line-numbers: true
    code-copy: true
    code-block-border-left: "#31BAE9"
    toc: true
    toc-depth: 1
    template-partials: 
      - ../toc-slide.html
    includes:
      in_header: ../sky_add_FilePointer.html
      after-body: ../toc-add.html
    toc-bg-image: "../../GrootbergGiraffe3.jpg"
lightbox: auto
filters:
  - parse-latex
include-before-body:
#### latex math shorthand
  file: ../MathShorthand.html 
include-in-header:
  - text: |
      <style>
       .title {
        font-size: 2.0em;
      }
      </style>
#### From https://github.com/quarto-dev/quarto-cli/discussions/2951
#### I am happy with the left-indenting of the section line
#### Horizontal centering can be challenging (!) in quarto: stupid
#### https://github.com/quarto-dev/quarto-cli/issues/1231
#### It turns out an easy solution is :::{style="text-align: center;"}
  - text: |
      <style>
      .center-xy {
        margin: 0;
        position: absolute;
        top: 50%;
        left: 20%;
        -ms-transform: translateY(-50%), translateX(-50%);
        transform: translateY(-50%), translateX(-50%);
      }
      </style>
bibliography: c:/seiro/docs/seiro.bib
#### ../../../seiro.bib
resources:
  - BPT.pdf
# quarto render BPT.qmd
---
```{css, echo = F, results = "hide"}
Red {
  color: red;
}
Orange {
  color: #e94e2c;
}
Blue {
  color: blue;
}
.reveal .slides pre code {
    max-height: 800px;
}
#TOC {
  font-color: #e94e2c;
}
.big-code{
  font-size: 110%  
}
#### Below is suggested by Claude
#### .small-font-list {
####   font-size: 70%;
####   line-height: 1.2;
#### }
#### .small-font-list {
####   font-size: 70%;
####   line-height: 1.2;
#### }
#### .small-font-list ol {
####   list-style-type: none;
####   padding-left: 0;
####   margin-left: 0;
####   counter-reset: item;
#### }
#### .small-font-list ol li {
####   margin-bottom: 0.5em;
####   display: flex;
#### }
#### .small-font-list ol li::before {
####   content: counter(item) ". ";
####   counter-increment: item;
####   margin-right: 0.5em;
####   flex-shrink: 0;
#### }
#### .small-font-list ol li > p:first-child {
####   margin: 0;
####   display: inline;
#### }
.small-font-list {
  font-size: 70%;
  line-height: 1.2;
}
.small-font-list ol {
  list-style-type: none;
  padding-left: 0;
  margin-left: 0;
  counter-reset: item;
}
.small-font-list ol li {
  counter-increment: item;
  display: flex;
  align-items: flex-start;
  margin-bottom: 0.5em;
}
.small-font-list ol li::before {
  content: counter(item) ". ";
  flex: 0 0 2.5em;  /* Adjust this value to change the width of the number area */
  text-align: right;
  margin-right: 0.5em;
}
#### description list by Claude
/* Main container for the description list */
.description-list {
  font-size: 50% !important;  /* Reduces font size to 70% of the parent element */
  line-height: 1.2 !important;  /* Sets line height to 1.2 times the font size */
}
/* Styles for the description list element */
.description-list dl {
  margin-left: 0;  /* Removes default left margin */
  padding-left: 0;  /* Removes default left padding */
}
/* Styles for the term (header) in the description list */
.description-list dt {
  font-weight: bold;  /* Makes the term text bold */
  display: inline;  /* Displays the term inline. Uncomment for a line break after item header. */
  margin-right: 0.5em;  /* Adds some space after the term */
}
/* Styles for the description in the description list */
.description-list dd {
  display: inline;  /* Displays the description inline. Uncomment for a line break after item header. */
  margin-left: 0;  /* Removes default left margin */
}
/* Styles for paragraphs within the description */
.description-list dd p {
  display: inline;  /* Keeps paragraph content inline. Uncomment for a line break after item header. */
  margin: 0;  /* Removes default paragraph margins */
}
/* Creates a new line and adds space between list items */
.description-list dd::after {
  content: "";  /* Necessary for the pseudo-element to render */
  display: block;  /* Creates a line break */
  margin-bottom: 0.5em;  /* Adds space between items. Adjust as needed */
}
#### description with a line break between item header and texts
.description-lb {
  font-size: 70%;
  line-height: 1.2;
}
.description-lb dl {
  margin-left: 0;
  padding-left: 0;
}
.description-lb dt {
  font-weight: bold;
  margin-bottom: 0.2em;
}
.description-lb dd {
  margin-left: 0;
  margin-bottom: 1em;  /* Adjust this value to control spacing between items */
}
.description-lb dd p {
  margin: 0;
}
```

@BarthPapageorgeThom2020

----

## SNP, polygenic score

::::: {layout="[10,-2,20]" layout-valign="bottom"}

::::{.fragment}
Genome

![Chromosone...DNA](DNAMacrostructure.jpg)

::: {style="font-size: 70%;"}
[https://courses.lumenlearning.com/...](https://courses.lumenlearning.com/suny-ulster-ap1/chapter/the-nucleus-and-dna-replication)
:::
::::

::::{.fragment}
![DNA structure](DNAStructure.jpg)

::: {style="font-size: 70%;"}
_[https://www.haibunda.com/...](https://www.haibunda.com/bundapedia/20230214003129-210-297249/deoxyribonucleic-acid--dna-)_
:::
::::

:::::

----

::::: {layout="[10,-2,20]" layout-valign="bottom"}

::::{.fragment}
![Double helix and rung](DoubleHelixRung.jpg)

::: {style="font-size: 70%;"}
[https://mammothmemory.net/...](https://mammothmemory.net/biology/dna-genetics-and-inheritance/dna-base-pairing/dna-double-helix.html)
:::
::::

::::{.fragment}
![SNP](SNP.png)

::: {style="font-size: 70%;"}
_[https://studylib.net/...](https://studylib.net/doc/5786293/snp-presentation)_
:::
::::

:::::

----

<!-- tree.nathanfriend.io -->

```{md}
.
└── Chromosome
    └── DNA
        └── Gene
            └── Genome
```


:::::{.columns}

::::{.column width="55%"}
![](genome.jpg)

::: {style="font-size: 70%;"}
[_Source: L. C. Douglas_](https://historiesofthingstocome.blogspot.com/2010/08/age-of-genome-4.html){height=5cm}
:::
::::

::::{.column width="45%"}
* Chromosomes: 1/2 each from both parents 
* DNA: Rolled up and shapes like an X
* Gene: DNA segments
* Genome: 3 billion nucletiodes (letters)
   * Letter sequence = genome sequence
   * 1.5 billion base pairs
::::

:::::

* Base pairs differ between people in less than 1% (15 million) locations  
* Single-nucleotide polymorphism (SNP) = such location
   * 2 chromosomes per person: AT-AT, GC-GC, AT-GC or 0, 2, 1 (reference = GC)

----

Twin studies

* Estimate how much genetic factors *collectively* matter for **explaining variations** of a trait  
   * Collectively: Unit = group level aggregation
   * Explaining variations: Variance decomposition 
* Do not reveal which SNP is correlated

. . .

Genome wide association studies (GWASs)

* Collect $J$ observable (?) SNPs of individual $i$.

. . .

Regress outcome $Y_{i}$ on each SNP using $J$ estimating equations:

$$
Y_{i}=\bfmu'\bfx_{i}+\beta_{j}SNP_{ij}+\epsilon_{ij}, \quad j=1,\dots,J.
$$ 

----

Polygenic score of $i$ for the outcome $Y$ = *"Educational attainment (EA) score"* 

$$
PGS_{i}=\sum_{j=1}^{J}\tilde{\beta} SNP_{ij}
$$

* Use Bayesian LDpred procedure to correct for correlations in $\tilde{\beta}_{j}$

* Use all SNPs: Better out-of-sample results than using only SNPs with genome-wide significance $p$ value $< 5*10^{-8} =$ .0000005%

* PGS is considered to be a predictor of individual fixed effects
   * HRS sample: covariance with EA score / total schooling years variance = 10.6%
   * Edu attainment SNPs $\propto$ biological process of brain development, cognition [@Okbay2016; @Lee2018]
   * Edu attainment SNPs $\propto$ cognition SNPs [@Okbay2016]


----

## Interpretation

* Correlation, not causality: Outcomes &lArr; gene + environment (endogenous)
* Attenuation due to measurement errors in $\hat{\beta}_{j}$
* Linearity assumption &rarr; underestimation of gene contributions (relative to twin studies)
* Population stratification: Ethnic-group fixed effects &lArr; gene, environment  
   * Including principal components of SNP data as covariates is shown to controll geographic variations
* External validity is up to HRS ancestral composition


----

## Data

Health and retirement survey: Age > 50 + partners, genetic samples 2006, 2008 

* All households with at least one individual with genetic European 
* "Retired households" in 1996, 1998, 2002, 2004, 2006, 2008, 2010
* Households with 1-2 members, drop same sex HHs
* Household-year observations of both members 65-75 years old

. . .

2590 HHs, 5701 HH-year observations (Table 1)

----



----
